scripts/push-debian.sh

```diff
@@ -63,21 +68,27 @@ fi
 
 [[ -z "${GCS_PATH}" ]] && [[ -z "${OUTPUT_DIR}" ]] && usage
 
+
+ARCH_NAME="k8"
+case "$(uname -m)" in
+  aarch64) ARCH_NAME="aarch64";;
+esac
+
 # Symlinks don't work, use full path as a temporary workaround.
 # See: https://github.com/istio/istio/issues/15714 for details.
 # k8-opt is the output directory for x86_64 optimized builds (-c opt, so --config=release-symbol and --config=release).
 # shellcheck disable=SC2086
-BAZEL_OUT="$(bazel info ${BAZEL_BUILD_ARGS} output_path)/k8-opt/bin"
+BAZEL_OUT="$(bazel info ${BAZEL_BUILD_ARGS} output_path)/${ARCH_NAME}-opt/bin"
 BAZEL_BINARY="${BAZEL_OUT}/tools/deb/istio-proxy"
 
 # shellcheck disable=SC2086
 bazel build ${BAZEL_BUILD_ARGS} --config=release ${BAZEL_TARGET}
 
 if [[ -n "${GCS_PATH}" ]]; then
-  gsutil -m cp -r "${BAZEL_BINARY}.deb" "${GCS_PATH}"/
+  gsutil -m cp -r "${BAZEL_BINARY}.deb" "${GCS_PATH}/istio-proxy${ARCH_SUFFIX}.deb"
```

Comment:
This changes name for x86 artifacts?

Reply:
No, since ARCH_SUFFIX will not be set for the x86 stuff for backwards compat